test/solc/CommandLineParser.cpp

```diff
@@ -224,6 +224,15 @@ BOOST_AUTO_TEST_CASE(cli_mode_options)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(no_append_metadata)
+{
+	vector<string> commandLine = {"solc", "--no-append-metadata", "contract.sol"};
+	CommandLineOptions parsedOptions = parseCommandLine(commandLine);
+	bool assert = parsedOptions.metadata.format == CompilerStack::MetadataFormat::NoMetadata;
+
+	BOOST_TEST(assert);
```

Comment:
```suggestion
	BOOST_TEST(parsedOptions.metadata.format == CompilerStack::MetadataFormat::NoMetadata);
```

Reply:
Surprisingly, this change leads to 10 pages of C++ errors. Some weird macro stuff going on here.

Reply:
Ah, ok then. I think it's because the macro does some magic that expects the type to be printable with `<<`. There are several ways around that. One, I think, was to put it in an extra parenthesis. You could also declare the type as non-printable with some macro. Or define the operator. But overall probably not worth the effort just for this one line. Let's leave it as is.